src/main/java/de/tum/in/www1/artemis/domain/iris/IrisMessage.java

```diff
@@ -43,6 +44,9 @@ public class IrisMessage extends DomainObject {
     @Enumerated(EnumType.STRING)
     private IrisMessageSender sender;
 
+    @Transient
+    private Integer nonce;
```

Comment:
What is this int used for? 
Also please provide a more descriptive name, as "number used only once" is not very helpful.

Reply:
This variable is used to help the client to differentiate between messages send from the current tab and all other tabs.